pointmatcher/TransformationsImpl.cpp

```diff
@@ -84,7 +84,7 @@ void TransformationsImpl<T>::RigidTransformation::inPlaceCompute(
 		const int descSpan(cloud.descriptorLabels[i].span);
 		const std::string& descName(cloud.descriptorLabels[i].text);
 
-		if (descName == "normals" || descName == "observationDirections")
+		if (descName == "normals" || descName == "observationDirections" || descName == "orientation_x" || descName == "orientation_y" || descName == "orientation_z")
```

Comment:
Descriptor names should follow the same case convention.

```suggestion
		if (descName == "normals" || descName == "observationDirections" || descName == "orientationX" || descName == "orientationY" || descName == "orientationZ")
```

Reply:
Thanks! I addressed it in the second commit.